clients/src/main/java/org/apache/kafka/clients/producer/internals/Sender.java

```diff
@@ -811,6 +811,13 @@ public void onComplete(ClientResponse response) {
         log.trace("Sent produce request to {}: {}", nodeId, requestBuilder);
     }
 
+    /**
+     * Send record error as error metric to sensor
+     */
+    public void sendError(String topic, int count) {
+        this.sensors.recordErrors(topic, count);
```

Comment:
In LKC we only expose the producer-topic-metrics https://jarvis.corp.linkedin.com/codesearch/result/?name=KafkaProducerSensor.java&path=linkedin-kafka-clients%2Fkafka%2Fkafka-client-container-impl%2Fsrc%2Fmain%2Fjava%2Fcom%2Flinkedin%2Fkafka%2Fclients%2Fsensors&reponame=multiproduct%2Flinkedin-kafka-clients#69, can you check this metric is under the producer-topic-metrics tag and not sender? Otherwise we'll need to expose a new metric for sender.